Formula/sdl2_sound.rb

```diff
@@ -0,0 +1,153 @@
+class Sdl2Sound < Formula
```

Comment:
Should this be called `sdl2_sound` given that the project is called `SDL_sound`?

Reply:
We already have an `sdl_sound` (for SDL1), and this follows the existing conventions from Homebrew and elsewhere.  (See `sdl2_image` for example.)